### PR TITLE
[WIP] Added feature files cs check on travis build

### PR DESCRIPTION
**Pull request by malukenho**
Added the feature files style check as discussed on #4006

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -72,6 +72,8 @@ script:
     - bin/phpspec run --no-interaction -f dot
     - bin/phpunit
 
+    - bin/kawaii gherkin:check --align=left features/
```
Comment (pamil) [inline .travis.yml]: It would be more consistent to have this check before the tests and just after the code CS checker.

File: composer.json [hunk 2]
```diff
@@ -100,7 +100,8 @@
         "matthiasnoback/symfony-dependency-injection-test": "^0.7.6",
         "phpspec/phpspec": "^2.4",
         "phpunit/phpunit": "~4.1",
-        "sensiolabs/behat-page-object-extension": "^2.0@dev"
+        "sensiolabs/behat-page-object-extension": "^2.0@dev",
+        "malukenho/kawaii-gherkin": "^0.1.2"
```
Comment (bendavies) [inline composer.json]: use `--sort-packages`

File: .travis.yml (hunk 1 above)
Comment (bendavies) [inline .travis.yml]: also only run on php7 to speed up the build.

**Comment by malukenho** [comment]
@pamil @bendavies thanks for the review. Done :smile:

**Comment by malukenho** [comment]
@pjedrzejewski rebased :+1:

**Comment by michalmarcinkowski** [comment]
@malukenho great job! Such a tool was missing! :+1:

**Pull request merged**